recipes/alm/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "ALM" %}
+{% set version = "2.0.0_dev.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/jochym/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 02aea008925df240d38cc23e08a14694a80186f4d7475464e337200132888ac4
+  patches:
+    - use_external_spglib.patch
+
+build:
+  number: 0
+  skip: True  # [win or py<35]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - eigen    
```

Comment:
Can you remove this?

Reply:
I am not sure I follow. eigen seems to be required for build (it is not enough to have it in host), and cxx is required for the c++ extension part of the package. But i will try if it builds without it.

Reply:
OK. It seems eigen is not required any more in build. So I am dropping it. Compiler of course is.